agenta-web/src/components/pages/settings/Secrets/Secrets.tsx

```diff
@@ -41,38 +50,52 @@ export default function Secrets() {
             </Text>
 
             <div className={classes.container}>
-                <Title level={5}>Providers</Title>
+                <Title level={5}>Providers API Key</Title>
```

Comment:
Can you please rename this to "Available Providers"? What do you say @mmabrouk ?

Reply:
sounds good

Reply:
I renamed it, honestly it sounded better